cmd/groups-auto-updater/test_files/test_peribolos_config.yaml

```diff
@@ -0,0 +1,39 @@
+orgs:
+  test-org:
+    admins:
+    - admin1
+    - admin2
+    - admin3
+    - admin4
+    members:
+    - member1
+    - member2  
+    billing_email: test@redhat.com
```

Comment:
nit: we don't need to set any fields we don't read, right? so just admins and members